<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AGNSS_LC29Y_Online_Dialog</class>
 <widget class="QDialog" name="AGNSS_LC29Y_Online_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>758</width>
    <height>357</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_5" columnstretch="6,3,1">
   <item row="0" column="0" rowspan="5">
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>FTP Client</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="FTP_Client" name="ftp_client" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QGroupBox" name="groupBox_utc">
     <property name="title">
      <string>UTC</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="2" column="0">
       <widget class="QLabel" name="label">
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string>Time accuracy:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="checkBox_useSysTime">
        <property name="text">
         <string>Use system time</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineEdit_timeAcy"/>
      </item>
      <item row="0" column="0">
       <widget class="QDateTimeEdit" name="dateTimeEdit"/>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEdit_leap_sec"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>leap second</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QGroupBox" name="groupBox_postion">
     <property name="title">
      <string>Postion</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Latitude</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEdit_lat">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="placeholderText">
         <string>degree</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Longitude</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLineEdit" name="lineEdit_long">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="placeholderText">
         <string>degree</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Altitude</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEdit_alt">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="placeholderText">
         <string>m</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Accuracy</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLineEdit" name="lineEdit_posAcy">
        <property name="placeholderText">
         <string>m</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>109</width>
          <height>15</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="2" colspan="2">
       <widget class="QCheckBox" name="checkBox_useCurPos">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Use Current Position</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QGroupBox" name="groupBox_file">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>110</height>
      </size>
     </property>
     <property name="title">
      <string>Data File</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QPushButton" name="pushButton_selectFile">
        <property name="maximumSize">
         <size>
          <width>41</width>
          <height>23</height>
         </size>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QLineEdit" name="lineEdit_filePath"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="1" colspan="2">
    <widget class="QProgressBar" name="progressBar_download">
     <property name="value">
      <number>24</number>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QComboBox" name="comboBox_restartMode">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <item>
      <property name="text">
       <string>None</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Cold start</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Warm start</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Hot start</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QPushButton" name="pushButton_transfer">
     <property name="text">
      <string>Transfer</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FTP_Client</class>
   <extends>QWidget</extends>
   <header location="global">ftp_client.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
